### ISSUE TITLE
Allow `jsonSortOrder` to be an object in the configuration

### ISSUE THREAD

**Issue by thraizz** [Gudahtt/prettier-plugin-sort-json#115]
Hi, thanks for the cool plugin!
We currently have no configuration files besides the configuration in package.json, but i would love to use the `jsonSortOrder` feature without creating a file that holds the configuration. Would that be possible? `jsonRecursiveSort` can be set to true already, so i thought it might be possible 😸 
Thanks again.

**Comment by Gudahtt** [comment]
I don't think this is supported by Prettier, unfortunately. Not as far as I could tell from their docs, and from digging into their codebase a bit. I would like this too!

**Comment by Gudahtt** [comment]
For reference, here is where the options are parsed in Prettier: https://github.com/prettier/prettier/blob/e4240aeaf03807fdc95284ab1a2900915b53136f/src/main/options-normalizer.js#L200

There is no "object" type, which is why I don't think this would work.

Though, we could consider accepting the sort configuration as a string. That might be better than using a separate file.

**Comment by Gudahtt** [comment]
I will update this configuration to use a JSON string rather than a JSON file. This should be easier to work with.

**Comment by Gudahtt** [comment]
@thraizz This was shipped in v1.0.0. Let me know if this works for you!

It's still not ideal I know. An object would be nicer. But at least it doesn't need a separate file anymore.

You could also maintain this as an object if your prettier config is in JavaScript. e.g.

```JavaScript
// in `.prettierrc.js`

const jsonSortOrder = {
  // ...
}

module.exports = {
  jsonSortOrder: JSON.stringify(jsonSortOrder),
}
```
